File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { EarthquakeCatalogSDK } from '..'


describe('exists', async () => {

  test('test-mode', () => {
    const testsdk = EarthquakeCatalogSDK.test()
    equal(testsdk instanceof EarthquakeCatalogSDK, true,
      'EarthquakeCatalogSDK.test() must return a client synchronously')
  })

})

File: rb/test/exists_test.rb
# EarthquakeCatalog SDK exists test

require "minitest/autorun"
require_relative "../EarthquakeCatalog_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = EarthquakeCatalogSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
